config/environments/production.rb

```diff
@@ -64,5 +64,23 @@
   config.active_record.dump_schema_after_migration = false
 
   Rails.application.routes.default_url_options[:host] =
-    'sarce-trainer-admin.herokuapp.com'
+    'www.sarce-trainer-admin.herokuapp.com'
+
+config.action_mailer.default_url_options = { host: "www.sarce-trainer-admin.herokuapp.com" }
+
+ config.active_support.deprecation = :notify
+
+ config.action_mailer.delivery_method = :smtp
+ onfig.action_mailer.raise_delivery_errors =false
```

Comment:
Set this to `true`: if there is a delivery error in production, it's important, so an error must be raised.